README.md

```diff
@@ -523,7 +524,7 @@ By default, FS crawler will exclude files starting with `~`.
 #### Indexing JSon docs
 
 If you want to index JSon files directly without parsing with Tika, you can set `json_support` to `true`.
-
+JSon contents will be stored directly under _source. If you need to keep JSon documents synchronized to the index, set option [Add as Inner Object](#add-as-inner-object) which stores additional metadata and the  JSon contents under field `object`.
```

Comment:
Really small change: can you change `the  JSon` to `the JSon` (one single space).